internal/frontend/show.go

```diff
@@ -261,3 +246,28 @@ func (f *Frontend) get(ctx context.Context, url string, v interface{}) error {
 
 	return nil
 }
+
+// Option allows to modify the frontend.
+type Option func(*ShowFrontend) error
```

Comment:
In the future will this be renamed to something like 'ShowOption' so that we can support having multiple of these type aliases? (i.e., one for ArtistOption, MovieOption, etc)